src/plugins/kibana_utils/public/render_complete/render_complete_dispatcher.ts

```diff
@@ -61,7 +63,7 @@ export class RenderCompleteDispatcher {
   public dispatchError() {
     if (!this.el) return;
     this.count++;
-    this.el.setAttribute('data-render-complete', 'false');
+    this.el.setAttribute('data-render-complete', 'true');
```

Comment:
👍

Reply:
@vadimkibana for followup work, we could accept an optional Error parameter, and use it to [update the contents of a `data-render-error` attribute](https://github.com/elastic/kibana/pull/114472) with a message. WDYT?